SecurityPkg/VariableAuthenticated/SecureBootConfigDxe/SecureBootConfigImpl.c

```diff
@@ -5308,6 +5396,26 @@ SecureBootCallback (
     File = NULL;
   }
 
+  if (Pk != NULL){
+    Pk = NULL;
+  }
+
+  if (PkCert != NULL){
+    PkCert = NULL;
+  }

```

Comment:
These don't do anything, and compiler will most likely optimize it away. Both are local variables that are running out of scope few lines later. They also don't hold any secrets, so purging isn't required (if they did, you would have to use `memset()`, or whatever UEFI lib variant of it is called, so the compiler doesn't optimize it). As these lines don't do anything, please remove them.